content/articles/a-sbert-based-flask-web-app-for-automatic-text-summarization/index.md

```diff
@@ -0,0 +1,204 @@
+There are a variety of ways to use SentenceTransformers in Python projects. Our preliminary work is described in this paper [Sentence-BERT:](https://arxiv.org/abs/1908.10084) Sentence Embeddings Using Siamese BERT-Networks. Using this framework, you may generate embeddings for sentences and texts in over 100 languages.

```

Comment:
The intro could have been better. It's not very beginner friendly.

The reader might not be familiar with these terms. Take a look at other articles to see how they've done.